src/main/webapp/conference.html

```diff
@@ -142,7 +142,10 @@ <h3 class="text-muted">WebRTC Conference</h3>
 <script>
 	var token = "<%= request.getParameter("token") %>";
 	var streamId = "<%= request.getParameter("streamId") %>";
-
+	var playOnly = "<%= request.getParameter("playOnly") %>";
```

Comment:
if playOnly value is something weird like "xdf2323",  then it will be regarded true? this is something be done on purpose

Reply:
This is not a problem because default usage is publish/play mode without parameter. So if one adds this wants to play without publish. If you want we can control only true value.

Reply:
fair enough